PokeStar/PokeStar/DataModels/Raid.cs

```diff
@@ -122,33 +125,52 @@ public bool PlayerAdd(SocketGuildUser player, int partySize, SocketGuildUser inv
             else
                return false;
          }
-         var newGroup = Groups.ElementAt(group).SplitGroup();
-         if (newGroup != null)
-            Groups.Add(newGroup);
-         CheckMergeGroups();
-         return true;
+
+         RaidGroup newGroup;
+         if (Groups.Count < GROUP_LIMIT)
```

Comment:
So what happens to the person who was added now that the group is over 20 people?